app/src/main/cpp/skyline/services/socket/bsd/IClient.h

```diff
@@ -70,11 +82,22 @@ namespace skyline::service::socket {
          */
         Result Connect(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
+        Result GetPeerName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        Result GetSockName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        Result GetSockOpt(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
```

Comment:
You should add a short comment for these.

Reply:
Don't bother, they're just Linux funcs anyway

Reply:
too late, but anyway it's not a problem for me

Reply:
Oneliner comments don't bother anyone and are really simple to add. IMO they're cool, helped me a ton initially when discovering the Skyline codebase, I think they're a nice touch.

Reply:
We kinda stopped doing them and it's pointless for socket funcs esp when there's full socket docs elsewhere